resources/views/manager/profile/about.html.twig

```diff
@@ -4,6 +4,7 @@
     Manager: Profile - About Me
 
 ============================================================================= #}
+{% set linkedInPattern = '^https:\/\/[a-z]{2,3}\.linkedin\.com\/.*$' %}
```

Comment:
This should be set somewhere central, like a config file or inside a validator, and passed in via the controller